go/vt/vtadmin/api.go

```diff
@@ -1126,6 +1126,7 @@ func (api *API) GetSchemaMigrations(ctx context.Context, req *vtadminpb.GetSchem
 		results = make([]*vtadminpb.SchemaMigration, 0, len(req.ClusterRequests))
 	)
 
+	m.Lock()
```

Comment:
Looks like this means that GetSchemaMigrations requests are serialized, right? And if one is taking a long time for whatever reason, they'll back up. Or am I misreading things?

Reply:
Resolving as on second look... I don't know why I thought this 😆 Each call has its own mutex and we're just synchronizing access to shared vars across the goroutines.

Reply:
yep!